.github/workflows/release.yaml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        inputs:
```

Comment:
Can you set the inputs as `required`?

Reply:
Added required: true property for all parameters.